apps/facilities/lib/repo.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Facilities.Repo do
+  @moduledoc """
+  Repo to get facilitiy information.
+  """
+  use RepoCache, ttl: :timer.hours(1)
+
+  def get_for_stop(stop_id) do
+    cache({stop_id}, fn {stop_id} ->
```

Comment:
comment(non-blocking): since it's just one argument, you don't need to wrap it in a Tuple.